ref/config/serverConfiguration.adoc

```diff
@@ -24,171 +18,134 @@ following example:
 </server>
 ----
 
-This example `server.xml` configures the server to support Java Server Pages 2.3,
-to listen to incoming traffic to localhost on port 9080, and to automatically
-expand WAR files when they are deployed.
+This `server.xml` file configures the server to do the following things:
 
-The `server.xml` is described in more detail <<server-xml,below>>. When
-discussing config for {projectName}, the term "server config" can be used to refer
-to all of the files that make up the server config, or specifically to the
-config held in XML files; it is usually clear in context and, if not, the
-phrase "server XML config" might be used to clarify. The next few sections
-describe each of the config files in the order they are processed.
+* Support the Java Server Pages 2.3 feature
+* Listen to incoming traffic to `localhost` on port `9080`
+* Automatically expand WAR files when they are deployed
 
-== server.env
-The `server.env` files are optional but, when present, are read by the
-`bin/server` script and specify environment variables primarily used to
-influence the behavior of the `bin/server` script. `server.env` files are read
-from the following locations in order:
+The term _server config_ can be used to refer to all of the files that make up the server configuration, or specifically to the configuration that's in the XML files. If it's not clear in context, you can use the term _server XML config_ to refer to the configuration in the XML files.
```

Comment:
The original version of this was describing how we use the term server config and server XML config in this document and elsewhere. Generally we use server config unless there is ambiguity. This changes it to be about the reader as a result it doesn't achieve the function it is intended to which is to explain terminology used in this document.

Reply:
Updated